.docker/src-cms/etc/periodic/1min/call-center-update

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+tokenFile="token.json"
+waitTimeFile="waittime.json"
+
+SERVICE_NAME=${1:-storage}
+
+if ! cf service "$SERVICE_NAME" >/dev/null 2>&1; then
+  echo "S3 service $SERVICE_NAME does not exist in space $SPACE"
+  exit 1
+fi
+
+SERVICE_KEY="storagekey-$SPACE-$SERVICE_NAME"
+
+if ! cf service "$SERVICE_NAME" >/dev/null 2>&1; then
+  echo "S3 service $SERVICE_NAME does not exist in space $SPACE"
+  exit 1
+fi
+
+# gather s3 credentials from storage key
+cf create-service-key $SERVICE_NAME $SERVICE_KEY
+export S3INFO=$(cf service-key $SERVICE_NAME $SERVICE_KEY)
+export S3_BUCKET=$(echo "$S3INFO" | grep '"bucket":' | sed 's/.*"bucket": "\([^"]*\)".*/\1/')
+# export S3_REGION=$(echo "$S3INFO" | grep '"region":' | sed 's/.*"region": "\([^"]*\)".*/\1/')
+# export AWS_ACCESS_KEY_ID=$(echo "$S3INFO" | grep '"access_key_id":' | sed 's/.*"access_key_id": "\([^"]*\)".*/\1/')
+# export AWS_SECRET_ACCESS_KEY=$(echo "$S3INFO" | grep '"secret_access_key":' | sed 's/.*"secret_access_key": "\([^"]*\)".*/\1/')
+
+# we might be running in circleci
+if [ -f /home/circleci/project/env.local ]; then
+  . /home/circleci/project/env.local
+fi
+# we might be running from a local dev machine
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+if [ -f $SCRIPT_DIR/env.local ]; then
+  . $SCRIPT_DIR/env.local
+fi
+if [ -f ./env.local ]; then
+  . ./env.local
+fi
+
+# yes I know there is a ton of repeated code here, I couldn't get functions to work properly
+if [ -e $tokenFile ]; then
+  echo "Token file exists."
```

Comment:
Once we have the rest of this nailed down, let's take another pass at these messages. I think we'll want to include a common string we can filter on, and clearly note ERROR versus INFO. Also, we might not need most of these at all.